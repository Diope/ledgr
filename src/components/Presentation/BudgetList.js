import React from 'react';

const BudgetList = (props) => {
  return ( 
    <div>
      <h1>My Budget</h1>
    </div>
   );
}
 
export default BudgetList;