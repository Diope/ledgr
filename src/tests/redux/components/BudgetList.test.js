import React from 'react';
import { shallow } from 'enzyme'
import BudgetList from '../../../components/Presentation/BudgetList'
import budgets from '../../fixtures/budgets';

describe('BudgetList component testing', () => {

  
  test('Should render the BudgetList component with fixture data', () =>{
    const wrapper = shallow(<BudgetList budgets={budgets}/>)
    expect(wrapper);
  });
});