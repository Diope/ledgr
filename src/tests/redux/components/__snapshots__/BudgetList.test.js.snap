// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BudgetList component testing Should render the BudgetList component with fixture data 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Connect(BudgetList)
    budgets={
      Array [
        Object {
          "amount": "42000",
          "createdAt": 0,
          "description": "Oh my",
          "id": "1",
          "note": "",
        },
        Object {
          "amount": "3921190",
          "createdAt": -172800000,
          "description": "Its a mechanical bull",
          "id": "2",
          "note": "Hello again",
        },
        Object {
          "amount": 850000000,
          "createdAt": 10364400000,
          "description": "Automobile",
          "id": "3",
          "note": "real estate",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": [Function],
    },
    "ref": null,
    "rendered": [Function],
    "type": Object {
      "$$typeof": Symbol(react.context),
      "_calculateChangedBits": null,
      "_context": Object {
        "$$typeof": Symbol(react.context),
        "Consumer": [Circular],
        "Provider": Object {
          "$$typeof": Symbol(react.provider),
          "_context": [Circular],
        },
        "_calculateChangedBits": null,
        "_currentRenderer": null,
        "_currentRenderer2": null,
        "_currentValue": null,
        "_currentValue2": null,
        "_threadCount": 0,
      },
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": [Function],
      },
      "ref": null,
      "rendered": [Function],
      "type": Object {
        "$$typeof": Symbol(react.context),
        "_calculateChangedBits": null,
        "_context": Object {
          "$$typeof": Symbol(react.context),
          "Consumer": [Circular],
          "Provider": Object {
            "$$typeof": Symbol(react.provider),
            "_context": [Circular],
          },
          "_calculateChangedBits": null,
          "_currentRenderer": null,
          "_currentRenderer2": null,
          "_currentValue": null,
          "_currentValue2": null,
          "_threadCount": 0,
        },
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
